supertokens_python/recipe/passwordless/recipe_implementation.py

```diff
@@ -229,20 +229,18 @@ async def update_user(self, user_id: str,
         return UpdateUserPhoneNumberAlreadyExistsErrorResult()
 
     async def delete_email_for_user(self, user_id: str, user_context: Dict[str, Any]) -> DeleteUserInfoResult:
-        user = await self.get_user_by_id(user_id, user_context)
-        if user is None:
-            return DeleteUserInfoUnknownUserIdErrorResult()
-        data = {'userId': user_id, 'email': None, 'phoneNumber': user.phone_number}
-        await self.querier.send_put_request(NormalisedURLPath('/recipe/user'), data)
-        return DeleteUserInfoOkResult()
+        data = {'userId': user_id, 'email': None}
+        result = await self.querier.send_put_request(NormalisedURLPath('/recipe/user'), data)
+        if result['status'] == 'OK':
+            return DeleteUserInfoOkResult()
+        return DeleteUserInfoUnknownUserIdErrorResult()
```

Comment:
The API has other statuses as well which should never happen. So the code here should reflect that. It should be something like:
```python
if result['status'] == 'OK':
   return DeleteUserInfoOkResult()
if result['EMAIL_ALREADY_EXISTS_ERROR']:
   raise Exception("Should never come here")
if result['PHONE_NUMBER_ALREADY_EXISTS_ERROR']:
   raise Exception("Should never come here")
return DeleteUserInfoUnknownUserIdErrorResult()
```

Same goes for the other delete function

Reply:
Done.